### DISCUSSION TITLE
Quarto with Neovim syntax highlighting for latex/math in qmd files

### DISCUSSION THREAD

**Discussion by rhaynes74** [quarto-dev/quarto-cli#8338] [Q&A]
### Description

Hi folks, I use neovim to edit my .qmd files. Although the markdown syntax is (colored) highlighted, math (inside $$ or $$ $$) does not get highlighted, it appears in the same color/font as the text around it.  Anyone have a configuration which highlights the math appropriately?

**Comment by mcanouil** [comment]
Probably better to search (and/or ask) on the Neovim extension repository: <https://github.com/quarto-dev/quarto-nvim>
